de.dlr.sc.virsat.model.extension.cefx.test/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2Test.java

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why do you commit an empty class? Where is the migrator gone? Without the class it won't work...